x-pack/plugins/stack_connectors/public/connector_types/jira/jira.test.tsx

```diff
@@ -142,7 +140,7 @@ describe('jira action params validation', () => {
         'subActionParams.incident.summary': [],
         'subActionParams.incident.labels': [],
         'subActionParams.incident.otherFields': [
-          `A maximum of ${MAX_OTHER_FIELDS_LENGTH} additional fields can be defined at a time.`,
+          `A maximum of ${MAX_OTHER_FIELDS_LENGTH} attributes can be defined at a time.`,
```

Comment:
I get what you mean with `attributes` but are we sure about this? @shanisagiv1 wanted specifically `additional fields` in the past in different places.

Reply:
Good point. I will revert it back to additional fields. I noticed that Jira has the same labels in the UI.

<img width="745" alt="Screenshot 2024-05-29 at 12 33 15 PM" src="https://github.com/elastic/kibana/assets/7871006/0b7886e1-c2ad-4643-b06e-2069d3248c12">